### PR TITLE
feat: zoom static scene

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: scripts/components/Main.scss [hunk 1]
```diff
@@ -30,6 +30,10 @@
   z-index: 4;
 }
 
+.h5p-editor-three-image-wrapper .h5p-three-image.scene-container {
+  overflow: hidden;
```
Comment (boyum) [inline scripts/components/Main.scss:35]: `overflow` is one of those properties where it's often hard to understand why it's used. Would it make sense to add an extra comment here explaining why we use it?

File: scripts/components/Scene/SceneTypes/StaticScene.js [hunk 2]
```diff
@@ -59,16 +77,42 @@ export default class StaticScene extends React.Component {
    * React life-cycle handler: Component did update.
    */
   componentDidUpdate() {
+    // Remove wheel event listener from prev scene
+    if (!this.props.isActive && this.props.sceneId === this.props.sceneWaitingForLoad) {
+      this.sceneWrapperRef.current?.removeEventListener('wheel', this.handleMouseWheel.bind(this), false);
```
Comment (boyum) [inline scripts/components/Scene/SceneTypes/StaticScene.js]: When we bind `this` to a function, we create a new function reference. Therefore, we don't remove the same reference that we added.

File: scripts/components/Scene/SceneTypes/StaticScene.js [hunk 3]
```diff
@@ -312,6 +380,324 @@ export default class StaticScene extends React.Component {
...
+    };
+
+    // Move sideways
+    const portrait = img.width < img.height;
+    const smallerThanOverlayWidth = img.width <= overlayElement.clientWidth;
+
+    if (portrait && smallerThanOverlayWidth) {
+      this.moveX = 0;
+    } else {
+      if (img.right + xDiff >= bounds.right && img.left + xDiff <= bounds.left) {
```
Comment (boyum) [inline scripts/components/Scene/SceneTypes/StaticScene.js]: Can we give these conditions names? It's a little hard to read the code now.

Comment (boyum) [reply scripts/components/Scene/SceneTypes/StaticScene.js]: The same goes for similar conditions elsewhere in the file.

File: scripts/components/Scene/SceneTypes/StaticScene.js [hunk 4]
```diff
@@ -312,6 +380,324 @@ export default class StaticScene extends React.Component {
...
+    }
+  }
+
+  /**
+   * Handle mouse down.
+   * @param {MouseEvent} event 
+   */
+  handleMouseDown(event) {
+    if (event.button !== 0) {
+      return; // Not left mouse button
```
Comment (boyum) [inline scripts/components/Scene/SceneTypes/StaticScene.js]: nitpick

```suggestion
    const isLeftMouseButton = event.button === 0;
    if (!isLeftMouseButton) {
      return;
```

File: scripts/components/Scene/SceneTypes/StaticScene.js [hunk 5]
```diff
@@ -396,6 +783,57 @@ export default class StaticScene extends React.Component {
     };
   }
 
+  /**
+   * Get adjusted position after image move or zoom.
+   * @param {number} posX X-coordinate.
+   * @param {number} posY Y-coordinate.
+   * @returns {object} Position with x and y.
```
Comment (boyum) [inline scripts/components/Scene/SceneTypes/StaticScene.js]:
```suggestion
   * @returns {{ posX: number; posY: number }} Position with x and y.
```

File: scripts/components/Main.js [hunk 6]
```diff
@@ -51,6 +51,8 @@ export default class Main extends React.Component {
       scoreCard: {},
       maxZoomedIn: false,
       maxZoomedOut: true,
+      zoomScale: 1,
+      updateZoomScale: false,
```
Comment (boyum) [inline scripts/components/Main.js]: Can we give a little explanation saying what this is used for?

```js
      /** ... */
      updateZoomScale: false,
```

File: scripts/components/Scene/SceneTypes/StaticScene.js (hunk 2 above)
Comment (henriettemoe) [reply scripts/components/Scene/SceneTypes/StaticScene.js]: Better now?

Comment (boyum) [reply scripts/components/Scene/SceneTypes/StaticScene.js]: Yes! 💐

**Pull request merged**